### PR TITLE
[1LP][RFR] updated test cases related to Multi-Region FA

### PR DESCRIPTION
**Pull request by izapolsk**

### CODE REVIEW

File: cfme/tests/cloud/test_instance_reconfigure.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        casecomponent: Infra
+        initialEstimate: 1/3h
+        testSteps:
+            1. Have a VM created in the provider in the Remote region which is
+               subscribed to Global.
+            2. Reconfigure the VM using the Global appliance.
+        expectedResults:
+            1.
+            2.
+            3. VM reconfigured, no errors.
```
Comment (john-dupuy) [inline cfme/tests/cloud/test_instance_reconfigure.py]: Q: should this be expecetedResult `3`?

File: cfme/tests/configure/test_config_manual.py [hunk 2]
```diff
@@ -122,6 +126,8 @@ def test_add_duplicate_subscription():
 
 
 @pytest.mark.manual
+@test_requirements.settings
+@test_requirements.multi_region
 @pytest.mark.tier(3)
 def test_add_bad_subscription():
     """
```
Comment (john-dupuy) [inline cfme/tests/configure/test_config_manual.py]: _Optional_ `caseposneg:` should be `negative` for this test. By default, I believe it's positive.

File: cfme/tests/configure/test_config_manual.py [hunk 3]
```diff
@@ -140,20 +146,20 @@ def test_add_bad_subscription():
...
+            5. Subscription hasn't been added. Add subscription task has failed
     """
     pass
 
 
 @pytest.mark.manual
+@test_requirements.settings
+@test_requirements.multi_region
 @pytest.mark.tier(3)
 def test_edit_bad_subscription():
```
Comment (john-dupuy) [inline cfme/tests/configure/test_config_manual.py]: Same comment about `caseposneg` here

File: cfme/tests/cloud/test_instance_reconfigure.py (hunk 1 above)
Comment (izapolsk) [reply cfme/tests/cloud/test_instance_reconfigure.py]: yeah, you caught me. fixing

**Pull request merged**